### PR TITLE
Fix etcd backup and restore for API v2 and v3

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://github.com/openshift/openshift-docs/pull/12629

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@juanluisvaladas, will you PTAL? I think that the note you want to add to the backup topic makes more sense if it's split in half, and this note was put in to address a specific bug:
> Because clusters upgraded from previous versions of {product-title} might contain v2 data stores, back up both v2 and v3 datastores.

I think your edit changes the intended meaning a bit too much.

File: day_two_guide/topics/proc_backing-up-etcd.adoc [hunk 1]
```diff
@@ -62,10 +65,9 @@ Before backing up etcd:
 * Ensure the proper certificates to connect to the etcd cluster.
 
 ifeval::["{context}" == "environment-backup"]
-. To ensure the etcd cluster is working, check its health.
-You can use both etcd v2 and v3 API versions as both commands backup the whole v2 and v3 data.
-Also the etcd nodes information is the same in API v2 and v3, therefore you can do health checks and member related
-operations indistinctly.
+. To ensure the etcd cluster is working, check its health. Because the etcd node
+information is the same with API v2 and v3, you can use either to run health
+checks and member-related operations.
```
Comment (juanluisvaladas) [inline day_two_guide/topics/proc_backing-up-etcd.adoc]: This is much better wording.  👍

File: day_two_guide/topics/proc_backing-up-etcd.adoc [hunk 2]
```diff
@@ -226,8 +228,8 @@ the `/var/lib/origin/openshift.local.etcd` directory.
 +
 [IMPORTANT]
 ====
-Because etcd backup copies the whole database both v2 and v3 are included either way
-in both methods.
+Because clusters upgraded from previous versions of {product-title} might
+contain v2 data stores, back up both v2 and v3 datastores.
```
Comment (juanluisvaladas) [inline day_two_guide/topics/proc_backing-up-etcd.adoc]: Actually the backup copies the whole databatase (both with etcdctl2 backup and etcdctl3 snapshot copy v2 **and** v3).

So it's really only necessary to backup one of the two. The thing is that a datastore might be empty, but if it has information it **certainly** be copied.

Comment (kalexand-rh) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @juanluisvaladas, I'm not an etcd expert, but the last information that I was given is that if you're using the v2 API but have newer info, you have to manually back up the etcd database, which is why the v2 instructions contain `# cp /var/lib/etcd/member/snap/db /backup/etcd-$(date +%Y%m%d)` and why we specifically call out backing up both of them.

**Comment by vikram-redhat** [comment]
@xltian - @wanghaoran1988  - bump. Could you please review?

**Comment by xltian** [comment]
@geliu2016  Will review from QE.

**Comment by geliu2016** [comment]
@kalexand-rh LGTM，thx

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL?

File: day_two_guide/topics/proc_backing-up-etcd.adoc [hunk 3]
```diff
@@ -224,6 +230,8 @@ the `/var/lib/origin/openshift.local.etcd` directory.
 ====
 Because clusters upgraded from previous versions of {product-title} might
 contain v2 data stores, back up both v2 and v3 datastores.
+
+If you do not have a v2 datastore, you can back up only the v3 datastore.
```
Comment (mburke5678) [inline day_two_guide/topics/proc_backing-up-etcd.adoc]: This sentence seems a bit odd, or obvious, to me. Maybe something like:
_Because clusters upgraded from previous versions of {product-title} might
contain v2 data stores, back up all etcd datastores, including any v2 and v3 datastores._

**Comment by mburke5678** [comment]
@kalexand-rh One nit. Otherwise, LGTM

File: day_two_guide/topics/proc_backing-up-etcd.adoc (hunk 3 above)
Comment (kalexand-rh) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: Thanks!

**Pull request merged**